packages/fxa-settings/src/pages/ResetPassword/ResetPasswordWithRecoveryKeyVerified/index.test.tsx

```diff
@@ -16,6 +16,10 @@ jest.mock('../../../lib/metrics', () => ({
   usePageViewEvent: jest.fn(),
 }));
 
+afterEach(() => {
```

Comment:
`fireEvent` in the metrics tests was calling all metrics for each test case.

Reply:
whoa, good catch